### PR TITLE
Add BitPay

### PR DESCRIPTION
**Pull request by mbezhanov**
Provides a BitPay Gateway config definition

See: https://jira.ilabsinc.com/browse/GWY-10874

### CODE REVIEW

File: spec/components/schemas/GatewayAccountConfig/BitPay.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    required:
+      - "credentials"
+    properties:
+      credentials:
+        type: object
+        description: BitPay credentials object
+        properties:
+          token:
+            type: string
+            description: BitPay merchant API token
```
Comment (colinbird) [inline spec/components/schemas/GatewayAccountConfig/BitPay.yaml]: should this have `format: password` ?

Comment (mbezhanov) [reply spec/components/schemas/GatewayAccountConfig/BitPay.yaml]: It appears as plain text in the BitPay UI, hence I left it as `type: string` on our end as well.

Comment (dara123) [reply spec/components/schemas/GatewayAccountConfig/BitPay.yaml]: is it a sensitive data? from the look of it it really similar our api key so I would think it's sensitive.

Comment (colinbird) [reply spec/components/schemas/GatewayAccountConfig/BitPay.yaml]: In the context of Rebilly, it's possible that a user may have access to the Gateway Accounts (for enabling downtime, changing websites etc) who may not have access to the keys.

Comment (mbezhanov) [reply spec/components/schemas/GatewayAccountConfig/BitPay.yaml]: This makes a lot of sense. I have updated the field definition.

**Pull request merged**